app/assets/javascripts/modules/dir_toggle.js

```diff
@@ -0,0 +1,50 @@
+// Module adds behavior to Preview/Close preview for file objects
+
+(function( global ) {
+  var Module = (function() {
+
+    return {
+      init: function() {
+        $('.sul-embed-container').each(function(){
```

Comment:
Please avoid using jquery for this as we are trying to eliminate jQuery as a dependency here.

Reply:
This is just a variation on `file_preview.js`. However, I got rid of all of the jquery except `.slideDown()` and `.slideUp()`. I'm open to suggestions on how to get rid of them, but since they are in use elsewhere not sure if it is desirable to have multiple slide implementations that may look different.